src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,68 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    private static final int MAX_SIZE = 20;
+    private static final int MIN_SIZE = 5;
+    private static final int DEFAULT_SIZE = 10;
+    private static final String DEFAULT_COLOR = Color.WHITE.getColor();
+    private final Random random = new Random();
+
+    public Figure getRandomFigureName() {
+        ColorSupplier colorSupplier = new ColorSupplier();
+        int index = random.nextInt(Forms.values().length);
+        switch (Forms.values()[index].getForm()) {
+            case "CIRCLE":
+                return makeCircle(random.nextInt(MAX_SIZE) + MIN_SIZE,
+                        colorSupplier.getRandomColor());
+            case "RECTANGLE":
+                int widht = random.nextInt(MAX_SIZE) + MIN_SIZE;
+                int height = random.nextInt(MAX_SIZE) + MIN_SIZE;
+                Figure rectangle = new Rectangle(widht, height);
+                rectangle.setColor(colorSupplier.getRandomColor());
+                rectangle.getArea();
+                rectangle.draw();
+                return rectangle;

```

Comment:
move all it to private method, do the same for others
```suggestion
                return prepareRectangle();
```